src/index.html

```diff
@@ -8,6 +8,101 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star">
+      </div>
```

Comment:
```suggestion
      <div class="stars__star"></div>
```

Reply:
don't move closing tag to new line in this case

Reply:
done